force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -6886,7 +6886,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Text to be display before the &apos;See Duplicate Contact&apos; button</shortDescription>
-        <value>See all duplicate Contacts found using matching rules. &lt;a href=&apos;https://powerofus.force.com/IP_Contacts&apos; target=&apos;_blank&apos;&gt;&lt;b&gt; Tell Me More &lt;/b&gt;&lt;/a&gt;</value>
+        <value>See all duplicate Contacts found using matching rules. &lt;a href=&quot;https://powerofus.force.com/IP_Contacts&quot; target=&apos;_blank&apos;&gt;&lt;b&gt; Tell Me More &lt;/b&gt;&lt;/a&gt;</value>
```

Comment:
@npsp-reedestockton Did this need to be changed? Saw the encoding changed from single to double quote but it's not changed for the `target` attribute

Reply:
> @npsp-reedestockton Did this need to be changed? Saw the encoding changed from single to double quote but it's not changed for the `target` attribute

@daniel-fuller It works fine as is.  The only reason for the encoding change was to pass it through the allow list filter in UTIL_HtmlOutput_CTRL.  The target attribute isn't considered in the allow list filter so it didn't need to change.  If it seems to weird to have the mix, we could change it to be consistent, but it isn't necessary.

Reply:
I had the same question, seems to work fine so no reason to change it.

Reply:
@daniel-fuller Consistency is a good thing.  I made the change you suggested.